core/sys/windows/kernel32.odin

```diff
@@ -179,6 +179,11 @@ foreign kernel32 {
 		lpNewFileName: LPCWSTR,
 		bFailIfExists: BOOL,
 	) -> BOOL ---
+	CopyFileA :: proc(
```

Comment:
As a rule, we only bind against Unicode-compatible APIs.

See for example the comment in `kernel32.odin`: 	`OutputDebugStringA :: proc(lpOutputString: LPCSTR) --- // The only A thing that is allowed`. There might be a few other exceptions that snuck in by mistake, but we're not adding more.

Reply:
1) Odin is built around Unicode support.
2) Consistency reduces bugs.
3) Why add footguns? The `A` variants of some APIs will take UTF-8 input if you're on the right version of Windows, but historically they don't.

@gingerBill might have additional reasons. He decided the rule.

Reply:
@Pingar5 

Odin does not use ASCII string, it is used UTF-8 strings. As such, we _require_ Unicode-compatible APIs.

Reply:
@Kelimion A-variants on windows only take UTF-8 strings if the application links itself against a manifest requesting utf-8 support, which Odin doesn't do by default anyway, so yeah, not that useful

Reply:
@flysand7 Not just that, it only works on some versions of Windows 10/11 too. Not all.

Reply:
@Pingar5 There are numerous ways depending on what you require.

If it's a string literal: `intrinsics.constant_utf16_cstring(...)` or `windows.L(...)` (`core:sys/windows`)

If it's a string value: `windows.utf8_to_wstring(s, allocator)` (`core:sys/windows`) https://github.com/odin-lang/Odin/blob/master/core/sys/windows/util.odin

Reply:
Okay, I removed the A functions from the PR. Let me know if there is anything else I need to change to get these merged in